contracts/RewardPool.vy

```diff
@@ -92,7 +92,7 @@ def _checkpoint_token():
     this_week: uint256 = t / WEEK * WEEK
     next_week: uint256 = 0
 
-    for i in range(20):
+    for i in range(40):
```

Comment:
Maybe I don't have the context, but based on the issue details in the report, it would be the same but with 40 weeks instead of 20. Right?
Please, confirm if that was the decision for the fix.

Reply:
40 weeks is more than enough.

No admin here so no state variable